modules/extension/tile-client/src/main/java/org/geotools/tile/Tile.java

```diff
@@ -171,8 +176,19 @@ public BufferedImage getBufferedImage() {
         }
     }
 
+    @Override
     public BufferedImage loadImageTileImage(Tile tile) throws IOException {
-        return ImageIOExt.readBufferedImage(getUrl());
+
+        Map<String, String> headers = new HashMap<>();
+        headers.put("User-Agent", "Geotools Http client");
+        try (InputStream is = setupInputStream(getUrl(), headers)) {
+            return ImageIOExt.readBufferedImage(is);
+        }
+    }
+
+    private InputStream setupInputStream(URL url, Map<String, String> headers) throws IOException {
+        HTTPClient client = new SimpleHttpClient();
```

Comment:
Wondering if SimpleHttpClient should default to the new user agent. It's not the first time I see cascading blocked because servers don't like the Java user agent.

Reply:
that might be an even better fix - I guess.